components/atoms/Search/search.stories.tsx

```diff
@@ -1,28 +1,69 @@
-import { ComponentStory } from "@storybook/react";
-import Search from "components/atoms/Search/search";
+import { Meta, StoryObj } from "@storybook/react";
```

Comment:
There's a lot of changes in this file because I converted it to the new Storybook format.